app/models/like.rb

```diff
@@ -1,2 +1,8 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_like_counter
+    post.increment!(:like_counter)
+  end
```

Comment:
- [OPTIONAL] As mentioned above, you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_like_counter method every time a new like has been saved to database